src/network/protocol/messages/getblocktxn.zig

```diff
@@ -0,0 +1,128 @@
+const std = @import("std");
+const protocol = @import("../lib.zig");
+
+const Sha256 = std.crypto.hash.sha2.Sha256;
+const BlockHeader = @import("../../../types/block_header.zig");
+const CompactSizeUint = @import("bitcoin-primitives").types.CompatSizeUint;
+const genericChecksum = @import("lib.zig").genericChecksum;
+/// GetBlockTxnMessage represents the "GetBlockTxn" message
+///
+/// https://developer.bitcoin.org/reference/p2p_networking.html#getblocktxn
+pub const GetBlockTxnMessage = struct {
+    block_hash: [32]u8,
+    indexes: []CompactSizeUint,
```

Comment:
in memory those are just regular u32. They only are converted to CompactUsize during serde

Reply:
u64 actually

Reply:
correct